include/IECoreHoudini/OBJ_SceneCacheTransform.h

```diff
@@ -105,7 +105,9 @@ class OBJ_SceneCacheTransform : public OBJ_SceneCacheNode<OBJ_SubNet>
 			UT_String shapeFilter;
 			UT_String tagFilterStr;
 			UT_StringMMPattern tagFilter;
+			UT_String tagGroupsPrefix;
```

Comment:
Oh, let's just hardcode it. I added it in case it was going to be used somewhere besides IE, but actually this is IECoreHoudini.